src/tests/ci.md

```diff
@@ -280,12 +277,12 @@ by listening for either Commit Statuses or Check Runs. Since the merge commit is
 based on the latest `master` and only one can be tested at the same time, when
 the results are green, `master` is fast-forwarded to that merge commit.
 
-Unfortunately testing a single PR at the time, combined with our long CI (~2
-hours for a full run), means we can’t merge too many PRs in a single day, and a
-single failure greatly impacts our throughput for the day. The maximum number of
+Unfortunately, testing a single PR at a time, combined with our long CI (~2
+hours for a full run), means we can’t merge enough PRs in a single day, and a
+single failure greatly impacts our throughput. The maximum number of
```

Comment:
Nit: I find the previous "too many" wording is more accurate than "enough"

Reply:
changing it to "a lot"